NotifyMe/services/service.py

```diff
@@ -0,0 +1,202 @@
+            return None
+        except Exception as e:
+            logger.error(f"General error: {e}")
+            return None
+        
+    def get_subscriptionId_data(self, data):
+        subscription = Subscription.objects.get(id=data.get('id'))
+        return subscription
+    
+    
+    def get_subscriptionDatabase(self): # returns dataabse to check is the database exists
+        return Subscription
+    
+    
+    
+
+class NotificationService:
+    def get_notification_data(self, request):
+        try:
+            notifications = Notification.objects.all()
+            return notifications
+        except ObjectDoesNotExist as e:
+            logger.error(f"Object does not exist error: {e}")
+            return None
+        except MultipleObjectsReturned as e:
+            logger.error(f"Multiple objects returned error: {e}")
+            return None
+        except DatabaseError as e:
+            logger.error(f"Database error: {e}")
+            return None
+        except Exception as e:
+            logger.error(f"General error: {e}")
+            return None
+
+    def get_notification_type_data(self, request):
+        try:
+            notification_types = NotificationType.objects.all()
+            return notification_types
+        except ObjectDoesNotExist as e:
+            logger.error(f"Object does not exist error: {e}")
+            return None
+        except MultipleObjectsReturned as e:
+            logger.error(f"Multiple objects returned error: {e}")
+            return None
+        except DatabaseError as e:
+            logger.error(f"Database error: {e}")
+            return None
+        except Exception as e:
+            logger.error(f"General error: {e}")
+            return None
+        
+    def get_notificationId_data(self, data):
+        notification = Notification.objects.get(id=data.get('id')) 
+        return notification
+    
+    def get_notificationDatabase(self):
+        return Notification
+    
+    
+    
+
+
+class SubscriptionPlanService:
+    def get_subscriptionPlan_data(self, request):
+        try:
+            subscription_plans = SubscriptionPlan.objects.all()
+            return subscription_plans
+        except ObjectDoesNotExist as e:
+            logger.error(f"Object does not exist error: {e}")
+            return None
+        except MultipleObjectsReturned as e:
+            logger.error(f"Multiple objects returned error: {e}")
+            return None
+        except DatabaseError as e:
+            logger.error(f"Database error: {e}")
+            return None
+        except Exception as e:
+            logger.error(f"General error: {e}")
+            return None
+        
+        
+    def get_subscriptionPlan_id(self, data):
+        subscriptionPlan = SubscriptionPlan.objects.all(id=data.get('id'))
+        return subscriptionPlan
+    
+    def get_subscriptionPlanDatabase(self):
+        return SubscriptionPlan
```

Comment:
why you need this

Reply:
to handle SubscriptionPlanModel.DoesNotExists exception

Reply:
this is not valid code ask chat gpt and review and say remove redudant code